npm test fails to run for some peculiar reason:
````
$ npm test

> gpii-grunt-lint-all@1.0.5 test E:\source\gits\gpii-grunt-lint-all
> node node_modules/nyc/bin/nyc.js node tests/all-tests.js

module.js:540
    throw err;
    ^

Error: Cannot find module 'infusion'
````

--- a/package.json
+++ b/package.json
@@ -30,6 +30,7 @@
         "grunt-jsonlint": "1.1.0",
         "grunt-lintspaces": "0.8.3",
         "grunt-markdownlint": "2.1.0",
+        "infusion": "3.0.0-dev.20180801T212157Z.09bf3d438",
         "json5": "1.0.1",
         "jsonlint": "1.6.3"
     },

